tools/footprint/footprint.xml

```diff
@@ -39,16 +39,16 @@
             input_dir=\$(python -c "import os; print(os.path.dirname('$bam_file'))")
 
             &&
-            mv \$(find \$input_dir -name "*.PARAM") PARAM
+            mv \$input_dir/*.PARAM PARAM
```

Comment:
The leading `\` should be removed. You want to use the cheetah variable, not a bash-ism.